yarn.lock

```diff
@@ -3861,6 +3861,7 @@ babel-preset-gatsby@^0.12.0:
     "@babel/plugin-proposal-nullish-coalescing-operator" "^7.12.1"
     "@babel/plugin-proposal-optional-chaining" "^7.12.1"
     "@babel/plugin-syntax-dynamic-import" "^7.8.3"
+    "@babel/plugin-transform-classes" "^7.12.1"
```

Comment:
I'm not entirely sure why these changed. Could you revert this file altogether and commit/push? You can revert it to the version on `develop` using the following git commands:

```
git fetch
git checkout origin/develop -- yarn.lock
git add yarn.lock
git commit -m "chore: revert the yarn.lock file"
```

Reply:
Jerel, your trick worked!!